firebase-storage/src/main/java/com/google/firebase/storage/StorageRegistrar.java

```diff
@@ -16,19 +16,31 @@
 
 import android.support.annotation.Keep;
 import android.support.annotation.RestrictTo;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.auth.internal.InternalAuthProvider;
 import com.google.firebase.components.Component;
 import com.google.firebase.components.ComponentRegistrar;
+import com.google.firebase.components.Dependency;
 import com.google.firebase.platforminfo.LibraryVersionComponent;
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
 
 /** @hide */
 @Keep
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class StorageRegistrar implements ComponentRegistrar {
   @Override
+  @Keep
```

Comment:
nit: no need to `@Keep` this method, just the class is enough

Reply:
Removed